packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/CreateView/CreateView.js

```diff
@@ -49,9 +46,6 @@ export function ReviewWorkflowsCreateView() {
       currentWorkflow: { data: currentWorkflow, isDirty: currentWorkflowIsDirty },
     },
   } = useSelector((state) => state?.[REDUX_NAMESPACE] ?? initialState);
-  const {
-    allowedActions: { canCreate },
-  } = useRBAC(permissions.settings['review-workflows']);
```

Comment:
The whole view is already wrapped in the `create` condition, so there is no need to repeat that in here.